Supporting Resources/v2.0/Datamodel/Datafiles/Activity I-SB-2-A.yml

```diff
@@ -34,7 +34,9 @@ longDescription:
 
   The automated process may require access to credentials and secrets required to build the software, such as the code signing certificate or access to repositories. Handle these with care. Sign generated artifacts using a certificate that identifies the organization or business unit that built it, such that its integrity and can be verified later.
 
-  Automation also simplifies including security checks to the build process. Implement static application security testing (SAST) to run as part of the build.
+  Finally, add relevant secuirty testing tools in the automated build pipeline to further leverage the automated pipeline for security.
```

Comment:
This sentence reads a bit weird to me, lot of words being repeated. Have you generalized SAST on purpose? If so, I'd still offer it as an example, as I think it's pretty common. Maybe sth like: 

"Finally, add appropriate automated security checks (e.g. using SAST tools) in the pipeline to leverage the automation for security benefit.

Reply:
Took your proposal.